#!/usr/bin/env python
PACKAGE = "head_manager"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("stimuli_discount_factor", double_t, 0, "Stimuli discount factor", 0.8, 0.1, 1.0)
gen.add("object_salience_factor", double_t, 0, "Salience factor (IsMovingToward fact for objects)", 1.0, 1.0, 15.0)
gen.add("head_salience_factor", double_t, 0, "Salience factor (IsMoving fact for agent heads)", 1.0, 1.0, 15.0)
gen.add("joint_salience_factor", double_t, 0, "Salience factor (IsMoving fact for agent joints)", 1.0, 1.0, 15.0)
gen.add("looking_salience_factor", double_t, 0 ,"Salience factor (IsLooking fact)", 1.0, 1.0, 15.0)

exit(gen.generate(PACKAGE, "head_manager", "ReactiveHeadMotion"))